### PR TITLE
Set Fieldset, Number Input, Password Input, Phone Input, Time Input and Toggle Button as stable components

### PR DESCRIPTION
**Pull request by minevala**
## Description
- Set Fieldset, File Input, Link, Number Input, Password Input, Phone Input, Time Input, and Toggle Button as stable components
- The decision of which components are stable is based on usage analytics, the release date, and the number of issues/feature requests

Closes #
- https://helsinkisolutionoffice.atlassian.net/browse/HDS-1005

## Motivation and Context
- It gives more confidence for HDS library users if there are fewer draft components, and we have forgotten to do this regularly

## How Has This Been Tested?
- locally on dev machine

### CODE REVIEW

**Review by VilleMiekkoja — APPROVED** [review]
Important update, thanks 👍

**Review by ronijaakkola — CHANGES_REQUESTED** [review]

File: site/docs/components/component_overview.mdx [hunk 1]
```diff
@@ -58,20 +58,20 @@ Component                                                        | Status
...
 [Footer](/components/footer)                                     | <StatusLabel type="alert">Draft</StatusLabel>        |                                                                                   | [<IconCheck/>](/storybook/react/?path=/story/components-footer--default)
 [Icon](/components/icons)                                        | <StatusLabel type="info">Stable</StatusLabel>        | [<IconCheck/>](/storybook/core/?path=/story/icons-icons--discord)                 | [<IconCheck/>](/storybook/react/?path=/story/icons-some--icondiscord)
-[Link](/components/link)                                         | <StatusLabel type="alert">Draft</StatusLabel>        | [<IconCheck/>](/storybook/core/?path=/story/components-link--default)             | [<IconCheck/>](/storybook/react/?path=/story/components-link--default)
+[Link](/components/link)                                         | <StatusLabel type="info">Stable</StatusLabel>        | [<IconCheck/>](/storybook/core/?path=/story/components-link--default)             | [<IconCheck/>](/storybook/react/?path=/story/components-link--default)
```
[the remark this answers is not in the document]
Comment (minevala) [reply site/docs/components/component_overview.mdx:65]: Ok, we can withdraw

File: site/docs/components/component_overview.mdx [hunk 2]
```diff
@@ -58,20 +58,20 @@ Component                                                        | Status
...
-[Fieldset](/components/fieldset)                                 | <StatusLabel type="alert">Draft</StatusLabel>        | [<IconCheck/>](/storybook/core/?path=/story/components-fieldset--default)         | [<IconCheck/>](/storybook/react/?path=/story/components-fieldset--default)
-[File input](/components/file-input)                             | <StatusLabel type="alert">Draft</StatusLabel>        |                                                                                   | [<IconCheck/>](/storybook/react/?path=/story/components-fileinput--single)
+[Fieldset](/components/fieldset)                                 | <StatusLabel type="info">Stable</StatusLabel>        | [<IconCheck/>](/storybook/core/?path=/story/components-fieldset--default)         | [<IconCheck/>](/storybook/react/?path=/story/components-fieldset--default)
+[File input](/components/file-input)                             | <StatusLabel type="info">Stable</StatusLabel>        |                                                                                   | [<IconCheck/>](/storybook/react/?path=/story/components-fileinput--single)
```
[the remark this answers is not in the document]
Comment (minevala) [reply site/docs/components/component_overview.mdx:62]: We have no users in analytics. We can withdraw this as well

File: site/docs/components/component_overview.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (minevala) [reply site/docs/components/component_overview.mdx]: - [x] Link moved back to draft

File: site/docs/components/component_overview.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (minevala) [reply site/docs/components/component_overview.mdx]: - [x] File Input moved back to draft

**Review by ronijaakkola — APPROVED** [review]

**Pull request merged**